command/commands.go

```diff
@@ -524,16 +514,49 @@ func Commands(metaPtr *Meta, agentUi cli.Ui) map[string]cli.CommandFactory {
 				Meta: meta,
 			}, nil
 		},
-		"operator keygen": func() (cli.Command, error) {
-			return &OperatorKeygenCommand{
+
+		// COMPAT(1.4.0): deprecated, remove in Nomad 1.5.0
+		// Note: we can't just put this in the DeprecatedCommand list
+		// because the flags have changed too. So we've provided the
+		// deprecation warning in the original command and when it's
+		// time to remove it we can remove the entire command
+		"operator keyring": func() (cli.Command, error) {
+			return &OperatorKeyringCommand{
 				Meta: meta,
 			}, nil
 		},
```

Comment:
I kind of want to move all the subcommands under `nomad operator keyring` but maybe that'll be a second step once we remove the deprecated commands and do https://github.com/hashicorp/nomad/issues/13381